src/cli/helpers_spec.js

```diff
@@ -1,87 +1,100 @@
 import { beforeEach, describe, it } from 'mocha'
 import { expect } from 'chai'
 import sinon from 'sinon'
-import { getTestCasesFromFilesystem } from './helpers'
+import { loadPicklesFromFilesystem } from './helpers'
 import { promisify } from 'bluebird'
 import EventEmitter from 'events'
 import fsExtra from 'fs-extra'
 import path from 'path'
 import PickleFilter from '../pickle_filter'
 import tmp from 'tmp'
+import { messages } from 'cucumber-messages'
+import { EventDataCollector } from '../formatter/helpers'
+import { uuid } from 'cucumber-messages/dist/src/IdGenerator'
```

Comment:
Yes, we shouldn't reach into `dist/*` for anything. If something isn't exposed that, we'll expose it though the default package export.

Reply:
I've seen this before, where editors like WebStorm will, when resolving imports for you, see the `.d.ts` file down in `dist` and give you that import instead of the entry point one. I've found some tooling that helps with this, testing it on a work project now and will get a PR up for messages later if it pans out.